### PR TITLE
Update Installing EF Core

### PR DESCRIPTION
**Pull request by ajcvickers**
This page is hopelessly out-of-date. There are other potential improvements we could make here, but at this time I'm just quickly updating it to be not completely wrong!

### CODE REVIEW

File: entity-framework/core/get-started/overview/install.md [hunk 1]
```diff
@@ -61,7 +54,7 @@ For more information, see [NuGet Package Manager Dialog](/nuget/tools/package-ma
 
 * To update the provider, use the `Update-Package` command.
 
-* To specify a specific version, use the `-Version` modifier. For example, to install EF Core 2.2.0 packages, append `-Version 2.2.0` to the commands
+* To specify a specific version, use the `-Version` modifier. For example, to install EF Core 2.2.0 packages, append `-Version 6.0.14` to the commands
```
Comment (ErikEJ) [inline entity-framework/core/get-started/overview/install.md:57]: => EF Core 6 packages

File: entity-framework/core/get-started/overview/install.md [hunk 2]
```diff
@@ -2,28 +2,21 @@
...
 ---
 # Installing Entity Framework Core
 
 ## Prerequisites
 
-* EF Core is a [.NET Standard 2.0](/dotnet/standard/net-standard) library. So EF Core requires a .NET implementation that supports .NET Standard 2.0 to run. EF Core can also be referenced by other .NET Standard 2.0 libraries.
-
-* For example, you can use EF Core to develop apps that target .NET Core. Building .NET Core apps requires the [.NET Core SDK](https://dotnet.microsoft.com/download). Optionally, you can also use a development environment like [Visual Studio](https://visualstudio.microsoft.com/vs), [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac), or [Visual Studio Code](https://code.visualstudio.com). For more information, check [Getting Started with .NET Core](/dotnet/core/get-started).
+* EF requires the most recent [.NET SDK](https://dotnet.microsoft.com/en-us/download).
+  * At runtime, EF Core requires a recent version of .NET --see [_EF Core releases)](xref:core/what-is-new/index) to find the minimal .NET version needed for the version of EF Core that you want to use.
```
Comment (roji) [inline entity-framework/core/get-started/overview/install.md:13]:
```suggestion
  * At runtime, EF Core requires a recent version of .NET - see [_EF Core releases)](xref:core/what-is-new/index) to find the minimal .NET version needed for the version of EF Core that you want to use.
```

File: entity-framework/core/get-started/overview/install.md [hunk 3]
```diff
@@ -35,7 +28,7 @@ To install or update NuGet packages, you can use the .NET Core command-line inte
   dotnet add package Microsoft.EntityFrameworkCore.SqlServer
   ```
 
-* You can indicate a specific version in the `dotnet add package` command, using the `-v` modifier. For example, to install EF Core 2.2.0 packages, append `-v 2.2.0` to the command.
+* You can indicate a specific version in the `dotnet add package` command, using the `-v` modifier. For example, to install EF Core 2.2.0 packages, append `-v 6.0.14` to the command.
```
Comment (roji) [inline entity-framework/core/get-started/overview/install.md]:
```suggestion
* You can indicate a specific version in the `dotnet add package` command, using the `-v` modifier. For example, to install EF Core 6.0.14 packages, append `-v 6.0.14` to the command.
```

**Review by roji — APPROVED** [review]
Yeah, agreed that this needs more attention, but good to do these basic fixes right away.

**Pull request merged**